api/src/middleware/__tests__/programsRouter.ts

```diff
@@ -63,5 +63,10 @@ describe('programsRouter', () => {
         done(err);
       });
     });
+
+    it('should respond with an internal server error if an error was thrown while listing programs', done => {
+      mockListProgramsWithActivities.mockRejectedValue(new Error());
+      appAgent.get('/').expect(500, done);
+    });
```

Comment:
I'm wondering if we want to test the not found error? 
Saw this in the docsRouter test file:
```
    it('should respond with a not found error if no doc exists with the given slug', done => {
      mockFindDocBySlug.mockResolvedValue(null);
      appAgent.get(`/test`).expect(404, done);
    });
```

Reply:
per the discussion in standup today, we'll keep this as is since there isn't a situation in which this request should return a 404 - it should either return a collection of programs and their activities, or an empty array.